picklefield/fields.py

```diff
@@ -78,24 +83,15 @@ def dbsafe_decode(value, compress_object=False):
     return loads(value)
 
 
-class PickledObjectField(models.Field):
-    """
-    A field that will accept *any* python object and store it in the
-    database. PickledObjectField will optionally compress its values if
-    declared with the keyword argument ``compress=True``.
-
-    Does not actually encode and compress ``None`` objects (although you
-    can still do lookups using None). This way, it is still possible to
-    use the ``isnull`` lookup type correctly.
-    """
+class BasePickledObjectField(models.Field):
```

Comment:
I'm open to split `PickledObjectField` into `BasePickledObjectField` and `PickledObjectField(BasePickledObjectField)` but I don't think support for `joblib` should live in this project. It should be easy enough to ship a package that depends on this one to define `JobLibPickledObjectField(BasePickledObjectField)`.